shiftcommerce/reference-site
# Make Search More Customisable

### Overview

Related Github issue: shiftcommerce/trendy-golf/issues/156

We had to make a few adjustments to the search page in shift-next as well as the search-bar in shift-react-components to allow us to implement these correctly in Trendys front-end. The search page can now accept custom Breadcrumb and ProductBlock components and the search-bar can now accept a custom placeholder. 

Although more work could be done to make these pages/components even more customisable, this work is enough to implement search in Trendy.

### QA

- The layout of both the search results and the search bar should not be broken.
- You should still be able to search from any page outside of checkout
